x-pack/plugins/fleet/common/authz.ts

```diff
@@ -144,8 +144,7 @@ export const calculateAuthz = ({
         // These are currently used by Fleet Server setup
         setup: fleet.all || fleet.setup,
         readEnrollmentTokens: (fleet.all || fleet.setup || fleet.agents?.all) ?? false,
-        readAgentPolicies:
-          (fleet.all || fleet.read || fleet.setup || fleet.agentPolicies?.read) ?? false,
+        readAgentPolicies: (fleet.all || fleet.setup) ?? false,
```

Comment:
why is `fleet.agentPolicies?.read` removed here?

Reply:
This is the case where the feature flag `subfeatureEnabled` is not enabled, so we should not check for subfeature privileges in that case

Reply:
ah I missed that, thanks